internal/planrun/ledger.go

```
@@ -0,0 +1,273 @@
+		return err
+	}
+	l.mu.Lock()
+	defer l.mu.Unlock()
+	if l.afterAppendLock != nil {
+		l.afterAppendLock()
+	}
+	f, err := os.OpenFile(filepath.Join(l.Dir, ledgerFile), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o600)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)